components/admin_console/user_autocomplete_setting/user_autocomplete_setting.tsx

```diff
@@ -4,14 +4,14 @@
 import React, {useCallback, useRef} from 'react';
 
 import {UserProfile} from '@mattermost/types/users';
-import {DispatchFunc} from 'mattermost-redux/types/actions';
 
-import GenericUserProvider from 'components/suggestion/generic_user_provider.jsx';
+import GenericUserProvider from 'components/suggestion/generic_user_provider';
 import Setting from 'components/admin_console/setting';
 import SuggestionBox from 'components/suggestion/suggestion_box';
 import SuggestionList from 'components/suggestion/suggestion_list.jsx';
+import {UserAutocomplete} from '@mattermost/types/autocomplete';
```

Comment:
can we rearrange this after `mport {UserProfile} from '@mattermost/types/users';` line